Directory.Build.props

```diff
@@ -22,6 +22,8 @@
     <NoWarn>$(NoWarn);NU5125</NoWarn>
     <!-- Suppress warnings about using SemVer 2.0. -->
     <NoWarn>$(NoWarn);NU5105</NoWarn>
+    <!-- Enables Strict mode for Roslyn compiler -->
+    <Features>strict</Features>
```

Comment:
Nit: can you put this in the propertygroup below that starts with the comment "Compilation options"?